html/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include avocado_result_html/resources *
```

Comment:
Re to Makefile, yes, we might clean it up a bit, but in general Makefile is a standardized way to deploy/treat/tackle/test/... projects. It's independent to python and easy to use. So I would like to keep it as a primary way to `deploy`, `install`, create `rpms` and to run `selfchecks`.